### DISCUSSION TITLE
Can fl_chart draw different colors with the condition whether the line is greater than zero.

### DISCUSSION THREAD

**Discussion by changyiyao** [imaNNeo/fl_chart#1465] [Q&A] [answered]
![image](https://github.com/imaNNeo/fl_chart/assets/766842/ed0a36af-7a0b-4de2-a7de-4619fef624d4)

As the image shows, it should be green when line above x-axis and should be red when the line is below x-axis.

**Comment by imaNNeo** [comment, marked as answer]
You can follow #1044
